call_handlers/debugtools_commands.lua

```diff
@@ -657,4 +657,50 @@ function Commands:set_enable_animation_text_command(session, response, bool)
    radiant.util.set_global_config('enable_effect_triggers', bool)
 end
 
+function Commands:exec_script_server(session, response, script, entity)
+   return self:exec_script(session, response, script, entity)
+end
+
+function Commands:exec_script_client(session, response, script, entity)
+   return self:exec_script(session, response, script, entity)
+end
+
+function Commands:exec_script(session, response, script, entity)
+   local expr, error
+   expr, error = loadstring('return ' .. script)
+   if not expr then
+      -- Maybe it's a statement?
+      expr, error = loadstring(script)
+   end
+   if error then
+      response:reject('ERROR: ' .. error)
+   end
+   
+   -- Set the entity as a global variable. This is terrible, but unavoidable if we want to
+   -- let executed code set global vars. We could instead compile the expression as a
+   -- function with an entity argument, but if it's a statement, it won't be able to
+   -- normally set global vars, so no REPL state.
+   local saved_entity = rawget(_G, 'e')
```

Comment:
I started off with "entity", but after playing with it for a while, it was a pain to type that every time. I considered using something particularly weird like $ and replacing it with some auto-generated variable name in the script before compiling, but that seems worse,